KsApi/models/User.swift

```diff
@@ -86,6 +87,7 @@ extension User: Argo.Decodable {
       <*> json <| "name"
       <*> User.NewsletterSubscriptions.decode(json)
       <*> User.Notifications.decode(json)
+      <*> json <|? "opted_out_of_recommendations"
```

Comment:
You can do:
`<*> json <| "opted_out_of_recommendations" <|> .failure(false)` to set `false` as a default value if parsing the key fails. Credit to @Scollaco for helping me figure out if Argo could do this 😄

Reply:
Just one thing... it should be `.success(false)`, since we want the decoder to consider it a successfully decoded property.